application/common/utilities.py

```diff
@@ -1,12 +1,12 @@
 from json import dumps, loads
 from os import environ, getenv
 from typing import Any, Dict, Union
-
 from aws_lambda_powertools import Logger
 from aws_lambda_powertools.utilities.data_classes.sqs_event import SQSRecord
 from boto3 import client
 
-logger = Logger(child=True)
+
+logger = Logger()
 
 
 def is_debug_mode() -> bool:
```

Comment:
This function can be deleted as it isn't used elsewhere

Reply:
Same with most functions in this file `is_mock_mode, get_environment_variable, invoke_lambda_function`